sdcm/nemesis.py

```diff
@@ -498,7 +498,6 @@ def disrupt_start_stop_major_compaction(self):
                              watch_for="User initiated compaction started on behalf of",
                              timeout=timeout,
                              stop_func=compaction_ops.stop_major_compaction)
-        self.tester.wait_no_compactions_running()
```

Comment:
Do i understand it correctly set it was not really needed all that time here?

Reply:
I introduced the wait logic when working on the feature tests for these compactions. The idea behind this was that I want to single out a specific compaction type and avoid the other compactions' making "noise" in the logs (which are watched for the signs of the compaction starting). This works well enough for the feature tests, but not when used as one of many nemesis with a stress load running. Hence, I'm moving this logic out of the nemesis classes and into the feature test module.

Reply:
It is still unclear, do we expect this nemesis work after removing the waiter?
Or that "noise" will make some unexpected/unpredictable effects?

Reply:
It should work without the waiter. Without the waiter the worst case should be a NoOp scenario - waiting for the give compaction to show up in the logs, hitting the timeout and doing nothing (as it will not trigger the "stop compaction" step).